VATSpy.dat

```diff
@@ -18326,8 +18326,10 @@ ZMUB|Ulaan Baatar|ZMUB|
 [UIRs]
 ADR_U|Adria Radar Upper|LJLA,LDZO,LQSB,LYBA,LWSS,LAAA
 AFRN|North Africa Control|DAAA,DRRR,DTTC,FTTT,GMMM,HLLL
+AFRC|Africa Central Control|FLFI,FNAN,FSSS,FWLL,FZZA,HKNA,HTDC,HUEC

```

Comment:
HBBA and HRYR missing.

Reply:
True but as HUEC already overlays these as it controls this airspace FL245+ they shouldn’t be needed.

Reply:
But HUEC is not defined to display over those FIR's. Since you have added AFRC as a UIR, it will say that you cover those airspaces above FL245 anyways instead of having to display a empty hole where it seems there is no control available.

Reply:
Understood. Thanks.